lib/data_update_scripts/20200415200651_index_reading_list_reactions.rb

```diff
@@ -0,0 +1,11 @@
+module DataUpdateScripts
+  class IndexReadingListReactions
+    def run
+      Reaction.readinglist.select(:id).in_batches do |batch|
+        Search::BulkIndexToElasticsearchWorker.set(queue: :default).perform_async(
+          "Reaction", batch.map(&:id)
```

Comment:
You can still call other AR methods on `ActiveRecord::Batches::BatchEnumerator` instances, so this should work and give you an id array right away, without going through objects:

```suggestion
          "Reaction", batch.pluck(:id)
```